src/domain/prompt/BasePrompt.ts

```diff
@@ -91,25 +72,29 @@ export abstract class BasePrompt<T extends IPromptConfig<T['value']>> implements
   }
 
   async fulfill(val: T['value'] | undefined): Promise<IRichCursorInputRequired | undefined> {
-    // allow prompt.fulfill() for continuation
```

Comment:
@bulatgab We should change this comment to say; "We need to exempt setting `this.value` when `prompt.fulfill()` is called without any arguments, because it would reset our state to "uninitialized" due to `val` being `undefined`."

Reply:
Thanks, Brett. It's clearer that way indeed. I will change